ui/src/app/pages/access/module.tsx

```diff
@@ -112,6 +117,10 @@ export const Module = (props: {
             <DARTitleComponent
               {...{ profile }}
               afterInitialClick={showRefresh}
+              onClick={() => {
+                setShowRefresh(true);
+                moduleAction();
+              }}
```

Comment:
All components that are clickable (the entirety of the module is clickable), the click action is handled via the ModuleBox's action property (lines 91-93). Since the IDENTITY module is not clickable in that way, I want the same functionality:

`setShowRefresh(true);
                moduleAction();`
                
to be called on the click of a button on the inside of DARTitleComponent. That is done via the onClick property. This is being done on lines 110-113 of the Identity Help Text:
https://github.com/all-of-us/workbench/pull/7879/commits/5ce43515ebb90fb26917fcac0ca576b86bf45896#diff-ca3dea5aa5f5fdc1aadf12bbfa9a876f076740e894c9ff89ff7e277300843624